### ISSUE TITLE
Is there a way to configure the colours?

### ISSUE THREAD

**Issue by ghistes** [naggie/dstask#110]
Hi,

I just tried to run the arm-binary on my phone in termux and it seems to work.

I can add tasks, however I cannot view them as all I get is a completely black square - there seem to be some issues with the terminal.

But also on my PC I am not entirely happy with the colours - is there a way to configure them or turn them off completely or do you have at least plans for such a feature?

**Comment by naggie** [comment]
Yes the feature is planned, see https://github.com/naggie/dstask/issues/75

It will either be via env var or a config file. 

That's strange -- is the termux temrinal black, and does it support 256 colours?

**Comment by ghistes** [comment]
I'll have a look at it later. I think termux is 256 colour, but maybe TERM is wrong...

I see you've already fixed the little bug with the "bad revision HEAD", it may also be worth documenting somewhere that it is not enough only to install git - you also have to configure at the very least email and name for it before you can use it. On most PCs git will already be configured but the ability to use it on a phone could be useful (it would be for me until a proper app is available), so it would be perfect if when running for the first time it would not only create the repo but makes sure that git is propery configured and maybe prompt for email and name to configure it, so that users are not tripped up.

**Comment by naggie** [comment]
That's a good point actually, please create an issue for it -- should be easy to implement.

**Comment by naggie** [comment]
DSTASK_COLOURS is most appropriate I think.
